hassio/src/components/hassio-card-content.ts

```diff
@@ -88,6 +119,34 @@ class HassioCardContent extends LitElement {
       ha-relative-time {
         display: block;
       }
+      .icon_image img {
+        max-height: 40px;
+        max-width: 40px;
+        margin-top: 4px;
+        margin-right: 16px;
+        float: left;
+      }
+      .grayscale {
+        filter: grayscale(1);
+      }
+      .dot {
+        position: absolute;
+        background-color: var(--paper-orange-400);
+        width: 12px;
+        height: 12px;
+        top: 8px;
+        right: 8px;
+        border-radius: 50%;
+      }
+      .blue-topbar {
```

Comment:
So if I am right, we're now losing the rounded corners on the top of the card when using this. Can we add a radius to the top left and right corners that match the card?